src/main.rs

```diff
@@ -37,6 +39,26 @@ enum Cli {
     StaticApi { dest: String },
     #[structopt(name = "show-person", help = "print information about a person")]
     ShowPerson { github_username: String },
+    #[structopt(name = "dump-teams", help = "Lists all teams")]
+    DumpTeams {
```

Comment:
Maybe rename this to `show-teams` since it's closer in output format to `show-person`?